extensions/dependencies/logic_proof.html

```diff
@@ -1,3 +1,4 @@
+<!-- TODO(gvishal): Add extension resource prefix here -->
```

Comment:
But, it would not be possible to do interpolation using angular. We cannot have script tags inside script tags.